builder/bundles.go

```diff
@@ -1181,3 +1187,65 @@ func getClosestAncestorOwner(path string) (int, int, error) {
 	gid := int(fi.Sys().(*syscall.Stat_t).Gid)
 	return uid, gid, nil
 }
+
+func writeOsReleaseFile(filename string, chrootDir string) error {
+	var f *os.File
+	err := os.MkdirAll(filepath.Dir(filename), 0655)
+	if err != nil {
+		return err
+	}
+	f, err = os.Create(filename)
+	if err != nil {
+		return err
+	}
+
+	err = f.Chmod(0644)
+	if err != nil {
+		return err
+	}
+
+	_, err = fmt.Fprintln(f, "NAME="+"\""+"Clear Linux OS"+"\"")
+	if err != nil {
+		return err
+	}
+	_, err = fmt.Fprintln(f, "ID="+"clear-linux-os")
+	if err != nil {
+		return err
+	}
+	_, err = fmt.Fprintln(f, "ID_LIKE="+"clear-linux-os")
+	if err != nil {
+		return err
+	}
+	_, err = fmt.Fprintln(f, "VERSION_ID="+"")
```

Comment:
In the case the os-release file is not found, its contents are created and written. Its then opened again to update the VERSION_ID. Instead can we just update the VERSION_ID here itself.
So maybe instead of calling `createOSReleaseFile()` inside `updateOSReleaseFile()`, we can call it inside `buildOsCore()`. Something like this:

```
	var filename string
	if b.Config.Mixer.OSReleasePath != "" {
		filename = b.Config.Mixer.OSReleasePath
	} else {
		filename = filepath.Join(chrootDir, "usr/lib/os-release")
	}

	if _, err := os.Stat(filename); err != nil {
		if os.IsNotExist(err) {
			if err = createOSReleaseFile(filename, version, chrootDir); err != nil {
				return errors.Wrap(err, "couldn't create os-release file")
			}
		} else {
			return err
		}
	} else {
		if err := updateOSReleaseFile(b, filename, version, chrootDir); err != nil {
			return errors.Wrap(err, "couldn't update os-release file")
		}
	}
```

Reply:
@reaganlo updated the createOSReleaseFile to update version_ID as well.
if createOSReleaseFile is successful, it return from function.
Still calling  createOSReleaseFile from UpdateOSReleaseFile.